Common/Securities/SecurityTransactionManager.cs

```diff
@@ -371,6 +415,17 @@ public IEnumerable<Order> GetOrders(Func<Order, bool> filter)
             return _orderProcessor.GetOrders(filter);
         }
 
+        /// <summary>
+        /// Gets all orders matching the specified filter. Specifying null will return an enumerable
+        /// of all orders.
```

Comment:
Same comment about None/null, maybe `IEnumerable<Order> GetOrders(Func<Order, bool> filter)` should have a default `null` value?